src/ucp/wireup/select.c

```diff
@@ -593,6 +606,20 @@ ucp_wireup_tl_iface_latency(ucp_context_h context,
     }
 }
 
+static int ucp_wireup_has_slow_lanes(ucp_wireup_select_context_t *select_ctx)
+{
+    ucp_wireup_lane_desc_t *lane_desc;
```

Comment:
space line after
ucs ucs_carray_for_each

Reply:
you mean after:
```
for (lane_desc = select_ctx->lane_descs;
         lane_desc < select_ctx->lane_descs + select_ctx->num_lanes;

Reply:
1. add space line after line 687
2. use ucs_carray_for_each instead of the for loop

Reply:
what's the advantage of using this macro?